src/trajectory/Interpolated.cpp

```diff
@@ -100,6 +100,18 @@ void Interpolated::evaluateDerivative(double _t, int _derivative,
     _tangentVector.setZero();
   }
 
+  //if(_t==getStartTime())
+  if(_t==mWaypoints[0].t)
+  {
+      const auto segmentTime = mWaypoints[1].t - mWaypoints[0].t;
```

Comment:
@mkoval  Could we assume that mWaypoints.size() is always >= 2 in an interpolated?